timescaledb/getting-started/create-hypertable.md

```diff
@@ -31,37 +32,25 @@ automatically based on the `chunk_time` and `chunk_size` parameters you specify.
 -- Hypertable to store weather metrics
 ----------------------------------------
 -- Step 1: Define regular table
-CREATE TABLE IF NOT EXISTS weather_metrics (
-
-   time TIMESTAMP WITHOUT TIME ZONE NOT NULL,
-   timezone_shift int NULL,
-   city_name text NULL,
-   temp_c double PRECISION NULL,
-   feels_like_c double PRECISION NULL,
-   temp_min_c double PRECISION NULL,
-   temp_max_c double PRECISION NULL,
-   pressure_hpa double PRECISION NULL,
-   humidity_percent double PRECISION NULL,
-   wind_speed_ms double PRECISION NULL,
-   wind_deg int NULL,
-   rain_1h_mm double PRECISION NULL,
-   rain_3h_mm double PRECISION NULL,
-   snow_1h_mm double PRECISION NULL,
-   snow_3h_mm double PRECISION NULL,
-   clouds_percent int NULL,
-   weather_type_id int NULL
+CREATE TABLE IF NOT EXISTS stocks_real_time (
+
+   time TIMESTAMPTZ NOT NULL,
+   symbol text NULL,
+   price double PRECISION NULL,
+   day_volume int NULL
 );
 
 -- Step 2: Turn into hypertable
-SELECT create_hypertable('weather_metrics','time');
+SELECT create_hypertable('stocks_real_time','time');
 ```
 
-Creating a hypertable is a two-step process. Start by using a `CREATE TABLE`
-statement to create a regular relational table. Then, use a `SELECT`
-statement with the `create_hypertable` function to convert the table into a
-hypertable. The `SELECT` statement requires the name of the table to convert,
-and the name of the time column in that table.
+Creating a hypertable is a two-step process:
+- Use a `CREATE TABLE` statement to create a regular relational table
+- Use a `SELECT` statement with the [`create_hypertable`][create-hypertable] function to convert 
+the table into a hypertable. The `create_hypertable` function requires two input 
+values, the name of the table, and the name of the time column.
```

Comment:
```suggestion
- Create a regular PostgreSQL table with `CREATE TABLE...`
- Convert it to a hypertable by calling the [`create_hypertable()`][create-hypertable] function to convert 
the table into a hypertable. The `create_hypertable()` function requires two input parameters - the name of the table, and the name of the time column.
```
I'm totally open to wording. For me, the "use... use" repeat just doesn't feel active.